custom_components/renault/translations/fr.json

```diff
@@ -4,7 +4,6 @@
       "kamereon_no_account": "Aucun compte Kamereon trouvé"
     },
     "error": {
-      "invalid_api_keys": "Clé API invalide",
       "invalid_credentials": "Authentification invalide"
```

Comment:
cf. comment for `en.json`

Reply:
I think the common strings are generated by HA core during the "publishing" time, so they cannot be used inside custom components.